primitives/storage/storage.go

```diff
@@ -31,7 +31,13 @@ func NextKey(key int64) int64 { panic("Not implemented!") }
 
 func Read(key int64, value_out int64, offset int32) int64 { panic("Not implemented!") }
 
-func Root() int64 { panic("Not implemented!") }
+func RootV2(key []byte) []byte {
+	psKey := utils.SliceToOffset(key)
```

Comment:
psKey -> keyOffsetSize ?
psValue -> valueOffsetSize ?
to match the naming in Get, Set, Blake256, ... functions.